matrix/lambdas/daemons/v0/driver.py

```diff
@@ -161,7 +162,7 @@ def _parse_line(line: str) -> str:
             return tokens[0]
 
         lines = data.splitlines()[1:]
-        return list(map(_parse_line, lines))
+        return list(collections.OrderedDict.fromkeys(map(_parse_line, lines)))
```

Comment:
This avoids the needs for importing collection and is easier to understand.

```suggestion
        return list(set(map(_parse_line, lines)))
```